### DISCUSSION TITLE
Ability to see build status on specific commit

### DISCUSSION THREAD

**Discussion by cristeahub** [community/community#43845] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Hello!

I love GitHub Mobile! It's an essential part of my development workflow.
We recently moved to GitHub Actions for our CI/CD needs and a part of that is deploy when anything is merged to `main`.

I love that on PRs I can easily see the current CI status. After moving to GitHub Actions, the experience has become even better.

I would love to be able to easily see what caused the last build error, or success on the latest commit on main. I can see if main is red or green, but I can not click on that icon (or somewhere else) to inspect the CI job that ran.

I request a link to the CI build from a commit, so that I can inspect why main is red without having to use the github.com webpage.

I would suggest letting the red X and the green checkmark to be clickable to see the CI status, as well as having a link to the CI build from the detail view of a single commit.

**Comment by eliperkins** [comment, marked as answer]
Hey @cristeahub ! That's great feedback. I agree that seeing Actions for specific commits would be awesome, especially now that [GitHub Mobile has support for PR Actions](https://github.blog/changelog/2022-10-04-introducing-actions-on-github-mobile/).

I've put together your feedback in our internal tracking repo for this, too. I'll follow up with the team and see how we can prioritize this. I'll keep you posted!

**Reply by ankitvgupta** [reply]
Along these lines, it would be awesome to just have a version of the "Actions" tab that's in the browser UI. 

Even if making the individual red X clickable on a particular commit will take some more UX development, just being able to scroll through all recent Actions in the mobile app would speed up my workflow a lot.

I end up going to the Github.com mobile Safari website which is definitely a step down from the iOS app experience.

**Reply by thesuperzapper** [reply]
@eliperkins I just want to share my support for adding the ability to see actions status for specific commits, as it makes the app effectively useless for debugging why a build failed.

So you end up having to open the website on mobile.
